modules/osd-aws-privatelink-firewall-prerequisites.adoc

```diff
@@ -262,8 +262,15 @@ If you do not use a default Red Hat NTP server, verify the NTP server for your p
 |Recommended. The FTP server used by `must-gather-operator` to upload diagnostic logs to help troubleshoot issues with the cluster.
 |443
 
-|`.osdsecuritylogs.splunkcloud.com`
-`inputs1.osdsecuritylogs.splunkcloud.com`
+|`*.osdsecuritylogs.splunkcloud.com`
+|Required. Used by the `splunk-forwarder-operator` as a logging forwarding endpoint to be used by Red Hat SRE for log-based alerting.
+|443 and 9997
+
+|`http-inputs-osdsecuritylogs.splunkcloud.com`
+|Required. Used by the `splunk-forwarder-operator` as a logging forwarding endpoint to be used by Red Hat SRE for log-based alerting.
```

Comment:
s/logging forwarding endpoint/log forwarding endpoint ?  Logging forwarding sounds odd to me. Might be right. I'm not sure.

Reply:
Good catch, @mburke5678! I've updated the phrase to 'log forwarding endpoint'.